vllm/distributed/kv_transfer/kv_connector/utils.py

```diff
@@ -4,10 +4,9 @@
 KV cache helper for store.
```

Comment:
ignore file, to be rebased once https://github.com/vllm-project/vllm/pull/26734 lands